RELEASING.adoc

```diff
@@ -0,0 +1,92 @@
+= Releasing Kiali UI
+
+This document covers the specifics of releasing Kiali UI, and is meant as a
+guide for Kiali release managers.
+
+The main document covering core can be found link:https://github.com/kiali/kiali/blob/master/RELEASING.adoc[here].
+
+== Requirements
+
+You must have write permissions to the public GitHub repository in order to be
+able to push the tags, but that's pretty much it. Most of our release process
+is done automatically by Travis.
+
+This document also uses 1.0.0 as the new version, just as an example.
+
+== Preparing for a new version
+
+After a release happens, the first step for a new "batch" of modifications is
```

Comment:
not sure how to fix this, but "after a release happens" does not sound like proper English. @JStickler ?

Reply:
Changed it to "After a release", what do you think?